L/libsixel/build_tarballs.jl

```diff
@@ -0,0 +1,40 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libsixel"
+version = v"1.8.6"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/saitoha/libsixel.git", "5db717dfef6fa327cd4025e7352550f63d20699c")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/libsixel/
+./configure --prefix=${prefix} \
+    --build=${MACHTYPE} \
+    --host=${target} \
+    --includedir=$WORKSPACE/destdir/include \
+    --libdir=$WORKSPACE/destdir/lib \
+    --enable-python=no
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms(; experimental=true)
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libsixel", :libsixel)
```

Comment:
There are also two executables, don't you care about them?
```suggestion
    LibraryProduct("libsixel", :libsixel),
    ExecutableProduct("img2sixel", :img2sixel),
    ExecutableProduct("sixel2png", :sixel2png),
```

Reply:
Those are for command-line tools for terminal users, I think we can safely drop them and mimic the functionality inside `ImageInTerminal` and with all other JuliaImages functions (e.g., `save`). For the purpose of `ImageInTerminal`, only the shared library is needed.

Reply:
Just to confirm if it's expected, so these executables are still shipped by libsixel_jll?

```julia
julia> using libsixel_jll

julia> run(`ls $(libsixel_jll.artifact_dir)/bin`)
img2sixel	libsixel-config	sixel2png
```

And the `products` part only controls how codes in the `_jll` package are generated?

Reply:
Correct